man/fi_domain.3.md

```diff
@@ -581,6 +583,17 @@ level features that are supported by the provider.
 
 The operational mode bit related to using the domain.
 
+## default_auth_key
+
+The default authorization key to associate with endpoint and memory 
+registrations created within the domain.
+
+## default_auth_keylen
+
+The length of the default authorization key for the domain. If set to 0, then
+there no authorization key will be associated with endpoints and memory 
+registrations created within the domain.
+
```

Comment:
An update should be made to the ep_attr auth_key fields to indicate that the domain auth_key will be used if one isn't provided.  Otherwise, the ep_attr auth_key will used instead.